packages/client/src/components/WelcomeBox/ImagePlaceHolder/ImagePlaceHolder.component.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const ImagePlaceHolder = () => {
```

Comment:
Similarly here, while having this in it's own component makes it easier to read the WelcomeBox code having this as it's own thing feels a bit "loud" since it cannot be used for anything else. 
It is possible to have more than one component in the same code file and **not** export it. So yo could move this into the same file as WelcomeBox to still make it easy to read but keeping it an internal detail.